src/ontology/edits/oeo-tasks.omn

```diff
@@ -0,0 +1,709 @@
+
+    
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000057>
+
+    
+ObjectProperty: OEO_00390026
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A relationship between a process and a plan specification, in which the process is predefined or planned by the plan specification.",
+        rdfs:label "structured by"@en
+    
+    SubPropertyOf: 
+        <http://purl.obolibrary.org/obo/RO_0000057>
+    
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000015>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/IAO_0000104>
+    
+    
+ObjectProperty: owl:topObjectProperty
+
+    
+Class: <http://purl.obolibrary.org/obo/BFO_0000015>
+
+    
+Class: <http://purl.obolibrary.org/obo/BFO_0000017>
+
+    
+Class: <http://purl.obolibrary.org/obo/BFO_0000035>
+
+    
+Class: <http://purl.obolibrary.org/obo/IAO_0000005>
+
+    
+Class: <http://purl.obolibrary.org/obo/IAO_0000007>
+
+    
+Class: <http://purl.obolibrary.org/obo/IAO_0000104>
+
+    
+Class: OEO_00020166
+
+    
+Class: OEO_00390002
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determination method for wind characteristics of a region is a methodology with the goal to understand the specific wind characteristics of a predefined region, encompassing data acquisition, modelling, and analytical procedures.",
+        rdfs:label "determination method for wind characteristics of a region"@en
+    
+    SubClassOf: 
+        OEO_00020166,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00390004
+    
+    
+Class: OEO_00390003
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Data acquisition and pre-processing is an action specification that describes steps to acquire raw data and how this data is cleaned, transformed, and structured for subsequent analysis.",
+        rdfs:label "data acquisition and pre-processing"@en
+    
+    SubClassOf: 
+        OEO_00390035
+    
+    
+Class: OEO_00390004
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determined wind characteristics of a region is an object specification that defines the goal of a methodology as a state of understanding the specific wind characteristics of a predefined region.",
```

Comment:
I might have missed the explanation of this in the issue: But what is the wind characteristic of a region? It might be a valid term on its own, but I feel like it could be explained / listed what attributes are subject to this: speed, ...?

Reply:
So far all of this has been more about the tasks that are part of the wind characteristics determination process. But I agree - it is a bit strange that we have no term that really explain many of the actual content of the tasks. Like what a wind characteristic is. Maybe we should add a few terms for that? Or at least expand the definitions?
@stap-m

Reply:
See also https://github.com/OpenEnergyPlatform/ontology/issues/1891#issuecomment-2434351853 😺  I think we need input from the domain experts here